HEADER

```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2021 Legacy Rewoven
```

Comment:
remove that header?

Reply:
Ack, so much changing.
![The Voices](https://c.tenor.com/L2yGz-RI-KYAAAAd/the-voices-meme.gif "AAAAAAAAAAAA")

Reply:
> Content length exceeded
https://tenor.com/view/the-voices-meme-cat-gif-23917781

Reply:
Finished